data/data/vsphere/bootstrap/outputs.tf

```diff
@@ -0,0 +1,3 @@
+output "bootstrap_ip" {
+  value = vsphere_virtual_machine.vm_bootstrap.default_ip_address
```

Comment:
Is this going to be a problem if this value is an empty string?

Reply:
No, it's not a problem. It is only used for the bootstrap gather. There will be code in a follow-up PR to handle querying the IP address from the vCenter (as we currently do) when this variable is empty.